src/rpdk/core/__init__.py

```diff
@@ -1,5 +1,5 @@
 import logging
 
-__version__ = "0.1.7"
+__version__ = "0.1.8"
```

Comment:
unrelated to the release, but we should really try to make a change to 1.x.0 at some point for versioning. doesn't really make sense for this product to be in the 0.1.x versioning almost a year after initial release.